File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Chts/CHT0110U00grdCHT0110ItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Chts
{
    [Serializable]
    public class CHT0110U00grdCHT0110ItemInfo
    {
        private String _sangCode;
        private String _sangName;
        private String _sangNameHan;
        private String _sangNameSelf;
        private String _startDate;
        private String _endDate;
        private String _bulyongYn;
        private String _sugaSangCode;
        private String _sugaSangName;
        private String _junyeomBuryn;
        private String _junyeomKind;
        private String _samangSangGroup;
        private String _rowState;

        public String SangCode
        {
            get { return this._sangCode; }
            set { this._sangCode = value; }
        }

        public String SangName
        {
            get { return this._sangName; }
            set { this._sangName = value; }
        }

        public String SangNameHan
        {
            get { return this._sangNameHan; }
            set { this._sangNameHan = value; }
        }

        public String SangNameSelf
        {
            get { return this._sangNameSelf; }
            set { this._sangNameSelf = value; }
        }

        public String StartDate
        {
            get { return this._startDate; }
            set { this._startDate = value; }
        }

        public String EndDate
        {
            get { return this._endDate; }
            set { this._endDate = value; }
        }

        public String BulyongYn
        {
            get { return this._bulyongYn; }
            set { this._bulyongYn = value; }
        }

        public String SugaSangCode
        {
            get { return this._sugaSangCode; }
            set { this._sugaSangCode = value; }
        }

        public String SugaSangName
        {
            get { return this._sugaSangName; }
            set { this._sugaSangName = value; }
        }

        public String JunyeomBuryn
        {
            get { return this._junyeomBuryn; }
            set { this._junyeomBuryn = value; }
        }

        public String JunyeomKind
        {
            get { return this._junyeomKind; }
            set { this._junyeomKind = value; }
        }

        public String SamangSangGroup
        {
            get { return this._samangSangGroup; }
            set { this._samangSangGroup = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public CHT0110U00grdCHT0110ItemInfo() { }

        public CHT0110U00grdCHT0110ItemInfo(String sangCode, String sangName, String sangNameHan, String sangNameSelf, String startDate, String endDate, String bulyongYn, String sugaSangCode, String sugaSangName, String junyeomBuryn, String junyeomKind, String samangSangGroup, String rowState)
        {
            this._sangCode = sangCode;
            this._sangName = sangName;
            this._sangNameHan = sangNameHan;
            this._sangNameSelf = sangNameSelf;
            this._startDate = startDate;
            this._endDate = endDate;
            this._bulyongYn = bulyongYn;
            this._sugaSangCode = sugaSangCode;
            this._sugaSangName = sugaSangName;
            this._junyeomBuryn = junyeomBuryn;
            this._junyeomKind = junyeomKind;
            this._samangSangGroup = samangSangGroup;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/chts/CHT0110U00grdCHT0110ItemInfo.java
package nta.med.data.model.ihis.chts;

import java.io.Serializable;
import java.util.Date;

public class CHT0110U00grdCHT0110ItemInfo implements Serializable {

	private String sangCode;
	private String sangName;
	private String sangNameHan;
	private String sangNameSelf;
	private Date startDate;
	private Date endDate;
	private String bulyongYn;
	private String sugaSangCode;
	private String sugaSangName;
	private String junyeomBuryn;
	private String junyeomKind;
	public CHT0110U00grdCHT0110ItemInfo(String sangCode, String sangName,
			String sangNameHan, String sangNameSelf, Date startDate,
			Date endDate, String bulyongYn, String sugaSangCode,
			String sugaSangName, String junyeomBuryn, String junyeomKind) {
		super();
		this.sangCode = sangCode;
		this.sangName = sangName;
		this.sangNameHan = sangNameHan;
		this.sangNameSelf = sangNameSelf;
		this.startDate = startDate;
		this.endDate = endDate;
		this.bulyongYn = bulyongYn;
		this.sugaSangCode = sugaSangCode;
		this.sugaSangName = sugaSangName;
		this.junyeomBuryn = junyeomBuryn;
		this.junyeomKind = junyeomKind;
	}
	public String getSangCode() {
		return sangCode;
	}
	public void setSangCode(String sangCode) {
		this.sangCode = sangCode;
	}
	public String getSangName() {
		return sangName;
	}
	public void setSangName(String sangName) {
		this.sangName = sangName;
	}
	public String getSangNameHan() {
		return sangNameHan;
	}
	public void setSangNameHan(String sangNameHan) {
		this.sangNameHan = sangNameHan;
	}
	public String getSangNameSelf() {
		return sangNameSelf;
	}
	public void setSangNameSelf(String sangNameSelf) {
		this.sangNameSelf = sangNameSelf;
	}
	public Date getStartDate() {
		return startDate;
	}
	public void setStartDate(Date startDate) {
		this.startDate = startDate;
	}
	public Date getEndDate() {
		return endDate;
	}
	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}
	public String getBulyongYn() {
		return bulyongYn;
	}
	public void setBulyongYn(String bulyongYn) {
		this.bulyongYn = bulyongYn;
	}
	public String getSugaSangCode() {
		return sugaSangCode;
	}
	public void setSugaSangCode(String sugaSangCode) {
		this.sugaSangCode = sugaSangCode;
	}
	public String getSugaSangName() {
		return sugaSangName;
	}
	public void setSugaSangName(String sugaSangName) {
		this.sugaSangName = sugaSangName;
	}
	public String getJunyeomBuryn() {
		return junyeomBuryn;
	}
	public void setJunyeomBuryn(String junyeomBuryn) {
		this.junyeomBuryn = junyeomBuryn;
	}
	public String getJunyeomKind() {
		return junyeomKind;
	}
	public void setJunyeomKind(String junyeomKind) {
		this.junyeomKind = junyeomKind;
	}
	
}
